client/packages.go

```diff
@@ -61,10 +62,21 @@ type ResultInfo struct {
 	SuggestedCurrency string `json:"suggested-currency"`
 }
 
+type ListOptions struct {
+	RefreshOnly bool
```

Comment:
Sorry, just a preliminary point before I forget. Will do a more complete review tomorrow.

Can we please call this `SelectRefresh`?  It's almost a nitpick, but it actually enables us to postpone the decision about what it means to mix that with other options (additive or exclusive).